Tensile/SolutionStructs.py

```diff
@@ -3214,6 +3214,24 @@ def assignDerivedParameters(state):
     state["LVCB"] = roundupRatio(state["LSCB"] , state["GlobalLoadVectorWidthB"])
     state["LVPB"] = roundupRatio(state["LSPB"] , state["GlobalLoadVectorWidthB"])
 
+    if state["SplitGlobalRead"] > 1:
+      if state["DirectToVgprA"] or state["DirectToVgprB"]:
+        reject(state, "SplitGlobalRead does not work with DirectToVgpr")
+      if state["GlobalReadCoalesceGroupA"]:
```

Comment:
Don't we need to check B?

Reply:
These parameters can be different for B, right?

Reply:
Aha yes you're right. I misread a section of the tensor code and thought they were hard-wired to the same value. Working on a fix now

Reply:
Fixed